manifests/c/CloudFoundry/CLI/v8/8.7.3/CloudFoundry.CLI.v8.installer.yaml

```diff
@@ -0,0 +1,49 @@
+# Created with YamlCreate.ps1 v2.3.2 $debug=QUSU.CRLF.7-4-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: CloudFoundry.CLI.v8
+PackageVersion: 8.7.3
+InstallerType: zip
+Installers:
+- Architecture: x64
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: cf.exe
+    PortableCommandAlias: cf
+  - RelativeFilePath: cf8.exe
+    PortableCommandAlias: cf8
+  InstallerUrl: https://s3-us-west-1.amazonaws.com/v8-cf-cli-releases/releases/v8.7.3/cf8-cli_8.7.3_winx64.zip
+  InstallerSha256: 972190130310BCADA5134E75B69C68A27E4C17ABF1F60ABB2D8B96E637293909
+- Architecture: x86
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: cf.exe
+    PortableCommandAlias: cf
+  - RelativeFilePath: cf8.exe
+    PortableCommandAlias: cf8
+  InstallerUrl: https://s3-us-west-1.amazonaws.com/v8-cf-cli-releases/releases/v8.7.3/cf8-cli_8.7.3_win32.zip
+  InstallerSha256: B858AD768178CF49857BDDBD1BC8FC33A414C604E86F8B2FFE8252E2816F4F7C
+- Architecture: x64
+  NestedInstallerType: inno
+  NestedInstallerFiles:
+  - RelativeFilePath: cf8_installer.exe
+  Scope: machine
+  InstallerUrl: https://s3-us-west-1.amazonaws.com/v8-cf-cli-releases/releases/v8.7.3/cf8-cli-installer_8.7.3_winx64.zip
+  InstallerSha256: F6732CE2C1D25F218C400A778ABA6C6EB1DB534D01787BD5983F40F6A4045993
+  AppsAndFeaturesEntries:
+  - DisplayVersion: v8.7.3
+    ProductCode: Cloud Foundry CLI_is1
+  ElevationRequirement: elevatesSelf
+- Architecture: x86
+  NestedInstallerType: inno
+  NestedInstallerFiles:
+  - RelativeFilePath: cf8_installer.exe
+  Scope: machine
+  InstallerUrl: https://s3-us-west-1.amazonaws.com/v8-cf-cli-releases/releases/v8.7.3/cf8-cli-installer_8.7.3_win32.zip
+  InstallerSha256: CDFE41D94DE0B3C8007F06857DFA201043B94DD95B155848F61986EB4219EAB1
+  AppsAndFeaturesEntries:
+  - DisplayVersion: v8.7.3
+    ProductCode: Cloud Foundry CLI_is1
+  ElevationRequirement: elevatesSelf
```

Comment:
```suggestion
```

Same feedback as https://github.com/microsoft/winget-pkgs/pull/136754#discussion_r1468808513. DisplayVersion is not `v8.7.3` here